packages/vue/src/pages/crm-contact-form/crm-contact-form.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="contact-form-container">
+    <div class="view-wrapper">
+      <dx-toolbar>
+        <dx-toolbar-item location="before">
+          <dx-button icon="arrowleft"></dx-button>
+        </dx-toolbar-item>
+        <dx-toolbar-item  location="before" :text="contactName"></dx-toolbar-item >
+        <dx-toolbar-item  location="after" locateInMenu="auto">
+          <dx-button
+              text="Terminate"
+              type="default"
+              stylingMode="contained"
+          ></dx-button>
+        </dx-toolbar-item >
+        <dx-toolbar-item  location="after" locateInMenu="auto"
+                          widget="dxDropDownButton"
+                          :options="{
+        text:'Action',
+      stylingMode:'text',
+      width:120,
+      items: ['Assign to Me', 'Archive']
+      }">
+
+        </dx-toolbar-item >
+        <dx-toolbar-item  location="after" locateInMenu="auto">
+          <div class="separator"></div>
+        </dx-toolbar-item>
+
+        <dx-toolbar-item
+            location="after"
+            locateInMenu="auto"
+            widget="dxButton"
+            showText="inMenu"
+            :options="{
+        text: 'Copy',
+        icon: 'copy'
+      }"
+        ></dx-toolbar-item>
+
+        <dx-toolbar-item
+            location="after"
+            locateInMenu="auto"
+            widget="dxButton"
+            showText="inMenu"
+            :options="{
+        text: 'Refresh',
+        icon: 'refresh',
+        onClick: refresh
+      }"
+        ></dx-toolbar-item>
+      </dx-toolbar>
+
+      <div class="panels">
+        <!--      <div class="left">
+                <contact-form [contactData]="contactData$"></contact-form>
+              </div>-->
+
+        <div class="right">
+          <!--        <contact-cards
+                      [contactName]="(contactData$ | async)?.name"
+                      [tasks]="(contactData$ | async)?.tasks"
+                      [activities]="(contactData$ | async)?.activities"
+                      [activeOpportunities]="activeOpportunities"
+                      [closedOpportunities]="closedOpportunities"
+                      [notes]="contactNotes"
+                      [messages]="contactMessages"
+                  ></contact-cards>-->
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {ref} from 'vue';
+
+import DxButton from 'devextreme-vue/button';
+import {
+DxToolbar ,
+    DxItem as DxToolbarItem,
+} from "devextreme-vue/toolbar";
```

Comment:
I think we need to replace these quotes with single quotes. And fix it in all application modules